src/components/StudentForm.jsx

```diff
@@ -39,21 +40,36 @@ const StudentForm = () => {
     });
     console.log(response);
     if (response.status === 201) {
-      setMessage('Student Added!');
+      alert('Student Added!');
     } else {
-      setMessage('Failed to add student!');
+      setMessage(response.message);
+      setShowMessage(true);
+      setTimeout(() => {
+        setShowMessage(false);
+      }, 2000);
     }
   };
 
   return (
     <>
-      <Button variant="primary" onClick={handleShow}>Add Student</Button>
+      <Button variant="primary" className='modal-btn'
+        style={{width: '66%', margin: 'auto', marginTop: '10px'}}
+        onClick={handleShow}>Add Student</Button>
 
       <Modal show={show} onHide={handleClose}>
         <Modal.Header closeButton>
           <Modal.Title>Student Entry Form</Modal.Title>
         </Modal.Header>
         <Modal.Body>
+          {
+            showMessage && (
+              <div className="alert-container">
+                <Alert variant="danger">
+                  {message}
+                </Alert>
+              </div>
```

Comment:
you use this same pattern a bunch of times, a good sign to extract it into its own component! [(DRY - don't repeat yourself](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)) 
```js
const Alert = ({ message, show, onClose, dismissible }) => {
   if (!show) { return null; }
   
   return (
      <div ... >
        <Alert .../>
       </div>
     );
  };
  ```
};